esmvaltool/utils/cmorizers/obs/cmorize_obs_Landschuetzer2016.py

```diff
@@ -0,0 +1,98 @@
+# pylint: disable=invalid-name
+"""ESMValTool CMORizer script.
+
+# #############################################################################
+  Landschuetzer 2016
+# #############################################################################
+
+  Tier
+     Tier 2: other freely-available dataset.
+
+  Source
+     https://www.nodc.noaa.gov/archive/arc0105/0160558/3.3/data/0-data/
+
+  Last access
+     20190308
+
+  Download and processing instructions
+     Download the following file in ${RAWOBS}/Tier2/Landschuetzer2016:
+      spco2_1982-2015_MPI_SOM-FFN_v2016.nc
+
+  Modification history
+     20190227-A_lova_to: written.
+
+# #############################################################################
+
+"""
+
+import logging
+import os
+
+import iris
+import numpy as np
+
+from .utilities import (_add_metadata,
+                        _fix_var_metadata,
+                        _fix_coords,
+                        _read_cmor_config,
+                        _save_variable)
+
+logger = logging.getLogger(__name__)
+
+# read in CMOR configuration
+CFG = _read_cmor_config('Landschuetzer2016.yml')
+
+
+def _fix_data(cube, var):
+    """Specific data fixes for different variables."""
+    logger.info("Fixing data ...")
+    # fix for bad missing value definition
+    cube.data = np.ma.masked_values(cube.data, cube.data.fill_value)
+    if var in ['fgco2', ]:
+        # Assume standard year 365_day
+        cube.data = cube.data * -12.01 / 1000. / (86400. * 365.)
```

Comment:
```suggestion
        cube = cube * -12.01 / 1000. / (86400. * 365.)
```
Don't realize cubes unnecessarily, also below

Reply:
I'll give it a try ... is the multiply operation going to be applied only to the data or also to other parts of the cube? see also https://github.com/ESMValGroup/ESMValTool/pull/942#discussion_r263763405

Reply:
iris tries to keep the correct metadata, so if you multiply by 10 I expect you will loose the name and the units, because if they were correct to begin with, they would no longer be correct after the multiplication, see also the [documentation](https://scitools.org.uk/iris/docs/latest/userguide/cube_maths.html). To keep the metadata, simply restore it after the multiply:
```python
metadata = cube.metadata
cube *= -12.01 / 1000. / (86400. * 365.)
cube.metadata = metadata
```

Reply:
I kind of suspected this, as I saw the same operation in CMIP5 fixes, e.g.
https://github.com/ESMValGroup/ESMValTool/blob/3e68d1593f544694618617368f8a3c88d0683f33/esmvaltool/cmor/_fixes/CMIP5/BNU_ESM.py#L44-L48

so why to put 3 lines of code instead of one that does exactly the same thing? Is there some caveat in using `cube.data`?

Reply:
Yes, it load all data into memory, see also the [documentation](https://scitools.org.uk/iris/docs/latest/userguide/real_and_lazy_data.html)

Reply:
That's fine. I modified the code accordingly.

Reply:
@bouweandela I checked the output file generated from 
`cube *= -12.01 / 1000. / (86400. * 365.)`
and they are the same as in the original file, like the operation was not applied at all (as in the previous attempt suggested by @valeriupredoi I mentioned earlier https://github.com/ESMValGroup/ESMValTool/pull/942#discussion_r263763405).

Reply:
my bad it was you and not valeriu !

Reply:
I just tried your script and it seems to work fine: the values of the `fgco2` variable in the output are order of magnitude smaller than in the original..

Reply:
I tested it again after the missing_value revision and data now looks ok ...